packages/mds-orm/tests/audits.ts

```diff
@@ -36,7 +36,6 @@ export default () =>
           .values(audits)
           .onConflict('DO NOTHING')
           .execute()
-        test.value(audits.length).is(records)
```

Comment:
Turns out TypeORM doesn't modify this variable at all, so nothing's really getting tested with this line.

Reply:
Actually, it does.... See: `id`:
```
BEFORE {
  audit_trip_id: '29372ccc-0a12-41f5-a723-d25ac713d00e',
  audit_subject_id: 'auditor@agency.city',
  audit_device_id: 'c2ceb989-aa84-45ac-904d-82b47fe1b172',
  provider_id: 'c8051767-4b14-4794-abc1-85aad48baff1',
  provider_name: 'ORM Test Provider',
  provider_vehicle_id: '79M-000000',
  provider_device_id: 'f7352027-c542-4ffd-866e-73e99ae2b888',
  timestamp: 1579630566176,
  recorded: 1579630566176
}
AFTER {
  audit_trip_id: '29372ccc-0a12-41f5-a723-d25ac713d00e',
  audit_subject_id: 'auditor@agency.city',
  audit_device_id: 'c2ceb989-aa84-45ac-904d-82b47fe1b172',
  provider_id: 'c8051767-4b14-4794-abc1-85aad48baff1',
  provider_name: 'ORM Test Provider',
  provider_vehicle_id: '79M-000000',
  provider_device_id: 'f7352027-c542-4ffd-866e-73e99ae2b888',
  timestamp: 1579630566176,
  recorded: 1579630566176,
  id: '1'
}
```

Reply:
So, you could test that the `id` column is _not_ `undefined` after the write.

Reply:
I could've sworn I printed it out and I didn't see the ID there. Hah.